### PR TITLE
Rust examples and documentation

### PR DESCRIPTION
**Pull request by thibault-martinez**
## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: documentation/docs/libraries/rust/getting_started.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+title: Getting Started With Rust
+description: Getting started with the official IOTA Client Library Rust library.
```
Comment (Thoralf-M) [inline documentation/docs/libraries/rust/getting_started.mdx]:
```suggestion
description: Getting started with the official IOTA Client Rust library.
```

File: documentation/docs/libraries/rust/getting_started.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+This example fetches node information
+
+```rust
+use iota_client::{Client, Result};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Create a client with that node.
+    let client = Client::builder()
+        .with_node("https://api.alphanet.iotaledger.net/")?
```
Comment (Thoralf-M) [inline documentation/docs/libraries/rust/getting_started.mdx]:
```suggestion
        .with_node("https://api.testnet.shimmer.network")?
```

File: examples/node_api_core/02_get_tips.rs [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
+        .with_node(&node_url)?
+        .with_node_sync_disabled()
+        .finish()?;
+
+    // Get tips.
     let tips = client.get_tips().await?;
 
-    // Prints the response.
-    println!("{:#?}", tips);
+    // Print the tips.
```
Comment (Thoralf-M) [inline examples/node_api_core/02_get_tips.rs]: Looks like a useless comment, also in the other examples

Comment (thibault-martinez) [reply examples/node_api_core/02_get_tips.rs]: Yeah I guess I wanted to comment every step but for prints maybe it's useless

Comment (thibault-martinez) [reply examples/node_api_core/02_get_tips.rs]: Removed

**Review by Thoralf-M — APPROVED** [review]

**Pull request merged**